crates/katana/primitives/src/genesis/test-genesis.json

```diff
@@ -0,0 +1,46 @@
+{
+	"number": 0,
+	"parentHash": "0x999",
+	"timestamp": 5123512314,
+	"stateRoot": "0x99",
+	"gasPrices": {
+		"ETH": 1111,
+		"STRK": 2222
+	},
+	"feeToken": {
+		"address": "0x55",
+		"name": "ETHER",
+		"symbol": "ETH",
+		"decimals": 18,
+		"class": "0x8"
+	},
+	"universalDeployer": {
+		"address": "0x77"
+	},
+	"allocations": {
+		"0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266": {
```

Comment:
Is there a purpose to use ethereum address here? Or it should be starknet addresses?

Reply:
meant to be starknet address lol